### PR TITLE
Added SCORM interactions support. See also adapt-contrib-spoor.

### PR DESCRIPTION
**Pull request by chris-steele**
https://adaptlearning.atlassian.net/browse/ABU-690

### CODE REVIEW

File: src/core/js/views/questionView.js [hunk 1]
```diff
@@ -320,6 +322,16 @@ define(function(require) {
 
         },
 
+        recordInteraction:function() {
+            var config = Adapt.config.get('_spoor');
```
Comment (oliverfoster) [inline src/core/js/views/questionView.js]: can we remove the spoor specific stuff from here? the framework should be agnostic., it should preferably do this check after the recordInteraction event

**Comment by oliverfoster** [comment]
did you still want the this.model.get("_recordInteraction") check from the model?

**Comment by chris-steele** [comment]
Yep - there'll be times when interactions will only be needed for certain questions. That's been the case in Yorkshire Water (ken15092) so decided the granularity of choice would be good.

**Comment by oliverfoster** [comment]
it'd make sense to have that bit here if you've put it in spoor.

**Comment by chris-steele** [comment]
Ah see what you mean, yes, agreed.

**Comment by oliverfoster** [comment]
+1 develop pr needed also

**Comment by taylortom** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Pull request merged**